src/musicbrainz/chromaprinter.cpp

```diff
@@ -10,6 +10,16 @@
 #include "util/samplebuffer.h"
 #include "util/performancetimer.h"
 
+// Types of fprint/encoded pointers take into account Chromaprint API version
+// (void* -> uint32_t*) and (void* -> char*) changed in v1.4.0 -- alyptik 12/2016
+#if (CHROMAPRINT_VERSION_MINOR > 3)
```

Comment:
You can not rely only on minor digit or you will have a problem when there will be a 2.0 version. 
Please use major too.